src/box/lua/net_box.c

```diff
@@ -839,9 +854,10 @@ netbox_send_and_recv_iproto(int fd, struct ibuf *send_buf,
 			}
 		}
 		size_t unused;
-		if (netbox_communicate(fd, send_buf, recv_buf,
-				       /*limit=*/required, /*boundary=*/NULL,
-				       /*boundary_len=*/0, &unused) != 0) {
+		if (netbox_communicate(fd, send_buf, on_send_buf_empty,
+				       recv_buf, /*limit=*/required,
+				       /*boundary=*/NULL, /*boundary_len=*/0,
```

Comment:
Not sure if we use such type of commenting...

Reply:
This was before my commit and i don't know should we fix this?

Reply:
Anyway this part traps into your diff, so why not fix it?